contracts/Oracles/ChainlinkPriceOracle.sol

```diff
@@ -6,24 +6,29 @@ import { Fixed256x18 } from "@tempusfinance/tempus-utils/contracts/math/Fixed256
 import { AggregatorV3Interface, IChainlinkPriceOracle } from "./Interfaces/IChainlinkPriceOracle.sol";
 import { BasePriceOracle } from "./BasePriceOracle.sol";
 
-abstract contract BaseChainlinkPriceOracle is BasePriceOracle, IChainlinkPriceOracle {
+contract ChainlinkPriceOracle is BasePriceOracle, IChainlinkPriceOracle {
     // --- Types ---
 
     using Fixed256x18 for uint256;
 
     // --- Constants & immutables ---
 
     AggregatorV3Interface public immutable override priceAggregator;
+    uint256 public immutable override DEVIATION;
```

Comment:
```suggestion
    uint256 public immutable override deviation;
```

Reply:
we can't because `PriceFeed` already depends on `DEVIATION` and this would break the interface

Reply:
Yeah, totally forgot about that.